packages/web-components/fast-components/src/accordion/accordion-item/accordion-item.template.ts

```diff
@@ -0,0 +1,46 @@
+import { html, ref } from "@microsoft/fast-element";
+import { endTemplate, startTemplate } from "../../patterns/start-end";
+import { AccordionItem } from "./accordion-item";
+
+export const AccordionItemTemplate = html<AccordionItem>`
+    <template
+        class="${x => (x.expanded ? "expanded" : "")}"
+        aria-expanded="${x => x.expanded}"
+        slot="item"
+    >
+        <div
+            class="heading"
+            part="heading"
+            role="heading"
+            aria-level="${x => x.headinglevel}"
+        >
+            <button
+                class="button"
+                part="button"
+                ${ref("button")}
+                aria-expanded="${x => x.expanded}"
+                aria-controls="${x => x.id}-panel"
+                id="${x => x.id}"
+                @click="${(x, c) => x.clickHandler(c.event as MouseEvent)}"
+            >
+                <span class="heading">
+                    <slot name="heading" part="heading"></slot>
+                </span>
+            </button>
+            ${startTemplate}
```

Comment:
It seems a *little* strange to me that the "start" and "end" are:
1. right next to each-other
2. not surrounding the heading slot

Can you help me understand why these two slots exist *here* in the template?

Reply:
They are laid out in a grid and with column definitions.

Reply:
Since it makes no difference I'll move the end template before the button, This was originally to put the elements above the button but I ended up using `z-index` anyway

Reply:
Actually I take that back, It's for focus order, since the content in `start` and `end` can be focusable we want the button to focus first, If we change the order the `start` content will focus first.